src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const historyClones = [];
+  const currentSt = { ...state };
```

Comment:
The variable name 'currentSt' is not descriptive enough. Consider renaming it to 'stateCopy' to clearly indicate that this is a copy of the state object.